ci/openshift-prow-rhcos.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Would it make sense instead to just check out the appropriate branch of `openshift/os` and call its `ci/build-test-qemu.sh`?

Reply:
Hmm, yes.  Would have to teach that script to handle the case where we're doing a PR on that repo versus when we're not but it seems doable.